src/arch/metal.rs

```diff
@@ -678,7 +678,6 @@ __bss_stop:
 static STACK_SIZE: usize = PLATFORM_STACK_SIZE;
 
 // ————————————————————————————— Context Switch ————————————————————————————— //
-
```

Comment:
We keep a space between a separator comment and the next line, so please do not delete that line, thanks!

Reply:
It's a mistake, I had no intention of changing it haha